blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,191 @@
+- **Node and OS components:** Operating system patches, security updates, and node-level components for both Linux and Windows nodes
+
+This holistic approach ensures that every layer of your full Kubernetes stack remains supported, secure, and stable throughout the entire LTS lifecycle.
+
+**Support Timeline Example:**
+- AKS LTS 1.28 (GA: Nov 2023): Supported until Feb 2026
+- AKS LTS 1.29 (GA: Mar 2024): Supported until Apr 2026
+- AKS LTS 1.30 (GA: Jul 2024): Supported until Jul 2026
+- Future LTS versions will follow the same 24-month support commitment from their respective GA dates of the AKS community versions.
+
+For the most up-to-date information on LTS version support timelines, please refer to the [AKS LTS calendar](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#lts-versions).
+
+
+Throughout the support lifecycle, AKS LTS versions receive:
+- Security patches and vulnerability fixes
+- Fixes for critical bugs that impact stability or functionality
+- Azure service integration updates
+- Compatibility updates for ecosystem tools and applications
+
+**Stay Informed:** You can track LTS patch rollouts and new LTS version availability in real-time using the [AKS Release Tracker](https://releases.aks.azure.com/webpage/index.html) under the Kubernetes version tab. This provides complete visibility into your LTS support lifecycle and upcoming releases.
+
+## Creating AKS LTS Clusters
+
+Creating an AKS cluster with LTS support is straightforward and uses the same familiar tools and processes you already know. Here's a simple example using Azure CLI:
+
+```bash
+# Create an AKS cluster with LTS support
+az aks create \
+    --resource-group myResourceGroup \
+    --name myAKSLTSCluster \
+    --tier LTS \
+    --kubernetes-version 1.29 \
+    --node-count 3
+```
+
+You can also deploy AKS clusters with LTS support using:
+- **Azure Portal:** Select "Long Term Support" option during cluster creation
+- **ARM Templates/Bicep:** Specify the LTS tier in your infrastructure-as-code deployments
+- **Terraform:** Updated AKS provider supports LTS cluster configuration
+
+**Important:** AKS LTS is available as part of the **Premium tier**. For detailed pricing information, please refer to the [AKS Premium tier pricing](https://azure.microsoft.com/pricing/details/kubernetes-service/).
+
+AKS clusters with LTS support maintain full compatibility with the existing AKS ecosystem, including Azure Monitor, Azure Policy, and all AKS-managed add-ons.
+
+**Industry-Leading Ecosystem Support:** Unlike other cloud providers that typically limit LTS guarantees to the base Kubernetes API, AKS LTS provides comprehensive support for popular add-ons and components that enterprises depend on for production workloads. This includes coordinated breaking change management, timely CVE fixes, and compatibility assurance throughout the LTS lifecycle.
+
+## AKS LTS Version Compatibility
```

Comment:
this doc does a good job on the positives of LTS and its value proposition. we should also add some lines/section on how should a user think about the pros/cons of being on community vs LTS. for example, community has less support window, but we also should cover the positives of customers regularly upgrading clusters (new functionality, less expensive than LTS) and also capture few investments we have on providing reliability while still taking up regular upgrades (planned maintenance,....). a reader of this doc will be leaning towards LTS yes, but have a scope here to layout our positioning and flowchart/thinking on when customers should choose which options of the full spectrum